runtime/lua/vim/loader.lua

```diff
@@ -21,7 +21,10 @@ local M = {}
 
 ---@alias LoaderStats table<string, {total:number, time:number, [string]:number?}?>
 
+---@nodoc
 M.path = vim.fn.stdpath('cache') .. '/luac'
+
+---@nodoc
```

Comment:
Let's mention `@nodoc` at `:help dev-lua-doc`. Also `@meta` . 

We don't need a complete list there, but the custom or unusual ones are worth mentioning.